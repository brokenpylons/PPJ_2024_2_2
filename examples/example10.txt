var grade1 = 6;
var grade2 = 7;
var grade3 = 8;
var grade4 = 9;
var grade5 = 10;
var sum = grade1 + grade2 + grade3 + grade4 + grade5;
var average = sum / 5;
print(average);
