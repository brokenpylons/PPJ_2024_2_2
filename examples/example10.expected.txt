lparen("(") real("2") plus("+") real("2") rparen(")") divides("/") real("2") 
